frontend/components/FieldDateTime/FieldDateTimeFilter.jsx

```diff
@@ -76,6 +76,8 @@ export default class FieldDateTimeFilter extends Component {
       valueStyles
     } = this.props
 
+    const format=(((config || {}).formats || {}).date || {}).short
```

Comment:
I understand why you're doing this, but I find this syntax super hard to read and not a justifiable price to pay for a single-liner. Could you refactor?